frontend/packages/kubevirt-plugin/src/components/vmtemplates/vm-template.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import * as classNames from 'classnames';
+import { sortable } from '@patternfly/react-table';
+
+import {
+  TemplateSource,
+  getTemplateOperatingSystems,
+  getTemplateFlavors,
+} from 'kubevirt-web-ui-components';
+
+import { ListPage, Table, TableRow, TableData } from '@console/internal/components/factory';
+import { Kebab, ResourceLink, ResourceKebab } from '@console/internal/components/utils';
+import { VmTemplateModel } from '../../models';
+
+export const menuActions = Kebab.factory.common;
+
+const { kind, selector } = VmTemplateModel;
+
+const tableColumnClasses = [
+  classNames('col-lg-2', 'col-sm-4', 'col-xs-4'),
```

Comment:
we could make constants out of these if we reuse them for multiple indexes

Reply:
Make sense to me too :+1: